File: FaceSample/app/src/main/java/com/regula/facesample/items/basic/LivenessDefaultItem.java
package com.regula.facesample.items.basic;

import android.content.Context;
import android.widget.Toast;

import com.regula.facesample.items.CategoryItem;
import com.regula.facesample.util.LivenessResponseUtil;
import com.regula.facesdk.FaceSDK;

/**
 * Created by Sergey Yakimchik on 20.08.21.
 * Copyright (c) 2021 Regula. All rights reserved.
 */

public class LivenessDefaultItem extends CategoryItem {

    @Override
    public void onItemSelected(Context context) {
        FaceSDK.Instance().startLiveness(context, livenessResponse -> LivenessResponseUtil.response(context, livenessResponse));
    }

    @Override
    public String getTitle() {
        return "Liveness";
    }

    @Override
    public String getDescription() {
        return "Detects if a person on camera is alive.";
    }
}

File: FaceSampleKotlin/app/src/main/java/com/regula/facesamplekotlin/items/basic/LivenessDefaultItem.kt
package com.regula.facesamplekotlin.items.basic

import android.content.Context
import com.regula.facesamplekotlin.category.CategoryItem
import com.regula.facesdk.FaceSDK
import com.regula.facesdk.model.results.LivenessResponse

class LivenessDefaultItem : CategoryItem() {

    override fun onItemSelected(context: Context) {
        FaceSDK.Instance().startLiveness(context) { livenessResponse: LivenessResponse? -> }
    }

    override val title: String
        get() =  "Liveness"

    override val description: String
        get() = "Detects if a person on camera is alive."
}